.travis.yml

```diff
@@ -68,21 +66,21 @@ before_install:
     fi
   - |
     # Configure mocha for testing
-    if   node_version_lt '0.10'; then npm_use_module 'mocha' '2.5.3'
+    if   node_version_lt '0.12'; then npm_use_module 'mocha' '2.5.3'
```

Comment:
Since you removed all less than 0.12 from the CI, this line can just be removed.